apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -220,9 +230,9 @@ class MediaPlayerController
 	void setUp(const int test)
 	{
 		mPlayer[0] = make_shared<MyMediaPlayer>();
-		mPlayer[0]->init(test);
+		mPlayer[0]->init(TEST_MP3);
```

Comment:
Why do you change this line?

Reply:
It just occurs during merging. Reverted.